### ISSUE TITLE
Pandas package

### ISSUE THREAD

**Issue by banerjeesoumya15** [kajarosz/climbing-routes-database#2]
Since you are reading/writing plane text from csv file, hence I suggest use [pandas](https://pandas.pydata.org/) package in python to read/write the csv files. This is more conventional way. No need to use loops.

**Comment by kajarosz** [comment]
Good point. I am planning on using pandas instead of csv module, but wanted to checkout how github works first. So far it's great and I am super happy that you contributed and raised this issue, thanks!  I will work on that today or tomorrow.
